test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetTrustTests.cs

```diff
@@ -0,0 +1,555 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.IO;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NuGet.Common;
+using NuGet.Configuration.Test;
+using NuGet.Test.Utility;
+using Test.Utility.Signing;
+using Xunit;
+
+namespace Dotnet.Integration.Test
+{
+    [Collection("Dotnet Integration Tests")]
+    public class DotnetTrustTests
+    {
+        private const string _successfulAddTrustedSigner = "Successfully added a trusted {0} '{1}'.";
+        private const string _successfulRemoveTrustedSigner = "Successfully removed the trusted signer '{0}'.";
+        private MsbuildIntegrationTestFixture _msbuildFixture;
+
+        public DotnetTrustTests(MsbuildIntegrationTestFixture fixture)
+        {
+            _msbuildFixture = fixture;
+        }
+
+        [Fact]
+        public void Trust_No_ActionCommand_DefaultTo_ListAction_Success()
```

Comment:
I renamed them. Can you check now?

Reply:
Not sure if I'm a bit picky about the names.. How about renaming the tests under DotnetTrustTests.cs into :  
DotnetTrust_ListInThisCondition_Succeed
DotnetTrust_AddInThisCondition_Succeed
DotnetTrust_RemoveInThisCondition_Succeed
So that you may filter those tests in the results by the same prefix `DotnetTrust_`, to see if all tests are correct.
And the test could be easily recognized where it's coming from.( dotnet integration tests, trust command)